internal/services/controller/util/util.go

```diff
@@ -0,0 +1,38 @@
+package util
+
+import (
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	KindPod           = "Pod"
+	ReasonOOMEviction = "Evicted"
+
+	AnnotationStarvedResource = "starved_resource"
+
+	ResourceMemory = "memory"
+)
+
+func IsOOMEvent(event *corev1.Event) bool {
+	if event.Reason != ReasonOOMEviction {
+		return false
+	}
+
+	if event.InvolvedObject.Kind != KindPod {
```

Comment:
Can there be OOM events for non pod kind? Should func name reflect that it checks for more? Unless it is moved to  that filter package.